docs/conf.py

```diff
@@ -45,6 +45,8 @@
     "sphinx_js",
     "autodocsumm",
     "sphinx_pyodide",
+    #    "sphinxarg.ext",
```

Comment:
I don't know where this comment is coming from. I don't have it in my local files.

Reply:
Nevermind. I was checking the wrong file locally. Twice.